polyglot/piranha/src/main.rs

```diff
@@ -26,7 +27,7 @@ fn main() {
 
   info!("Executing Polyglot Piranha");
 
-  let args = PiranhaArguments::from_command_line();
+  let args = PiranhaArguments::from(PiranhaInput::CommandLineInput);
```

Comment:
`PiranhaInput::CommandLineInput` here at some level represents the side effect. of parsing from command line.